be/src/env/env_stream_pipe.cpp

```diff
@@ -25,8 +25,9 @@ Status StreamPipeSequentialFile::read_one_message(std::unique_ptr<uint8_t[]>* bu
     return _file->read_one_message(buf, length, padding);
 }
 
-Status StreamPipeSequentialFile::skip(uint64_t n) {
-    return _file->seek(n);
+Status StreamPipeSequentialFile::skip(int64_t n) {
+    std::unique_ptr<char[]> buf(new char[n]);
+    return read(buf.get(), n).status();
```

Comment:
Need to ensure that `n` is actually read.

Reply:
Good finding!